### PR TITLE
Add export credit balance to returns journey

### PR DESCRIPTION
**Pull request by lmcewanhmrc**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -35,7 +35,7 @@ obligationPeriod.caption = April to June 2022
 timeout.title = You’re about to be signed out
 timeout.message = For security reasons, you will be signed out of this service in
 timeout.keepAlive = Stay signed in
-timeout.signOut = Sign out
+timeout.signOut = Sign out\\
```
Comment (liampan) [inline conf/messages.en:38]: boo

File: app/controllers/ConvertedPackagingCreditController.scala [hunk 2]
```diff
@@ -44,34 +49,60 @@ class ConvertedPackagingCreditController @Inject() (
...
+    exportCreditsConnector.get(request.request.pptReference,
+      obligation.fromDate.minusYears(2),
+      obligation.fromDate.minusDays(1)
+    )
+  }.map {
+    case Right(balance) => Some(displayMonetaryValue(balance.totalExportCreditAvailable))
+    case Left(_) => None
+  }
+
+  def displayMonetaryValue(money: BigDecimal): String = s"£${money.setScale(2, RoundingMode.HALF_EVEN)}"
```
Comment (liampan) [inline app/controllers/ConvertedPackagingCreditController.scala:84]: this is already a method on bigdecial in the viewmodel package i believe `moneyt.asPounds`

File: conf/messages.en (hunk 1 above)
Comment (C-Thomp) [inline conf/messages.en:38]: Missing comment?

**Review by C-Thomp — APPROVED** [review]
👍

File: conf/messages.en (hunk 1 above)
Comment (lmcewanhmrc) [reply conf/messages.en:38]: Eh? Er... how'd that happen? doh :D

File: app/controllers/ConvertedPackagingCreditController.scala (hunk 2 above)
Comment (lmcewanhmrc) [reply app/controllers/ConvertedPackagingCreditController.scala:84]: Fair enough. This line is just copied from 'other' branch

File: conf/messages.en (hunk 1 above)
Comment (lmcewanhmrc) [reply conf/messages.en:38]: Dunno where that came from? :D

Comment (lmcewanhmrc) [reply conf/messages.en]: should be fixified, good spot thanks

File: app/controllers/ConvertedPackagingCreditController.scala (hunk 2 above)
Comment (lmcewanhmrc) [reply app/controllers/ConvertedPackagingCreditController.scala]: sorted

File: conf/messages.en (hunk 1 above)
Comment (lmcewanhmrc) [reply conf/messages.en]: sorted, thanks :)

File: test/forms/ConvertedPackagingCreditFormProviderSpec.scala [hunk 3]
```diff
@@ -26,7 +26,7 @@ class ConvertedPackagingCreditFormProviderSpec extends DecimalFieldBehaviours {
 
   val packagingCredit = 1.25
   val formProvider            = new ConvertedPackagingCreditFormProvider
-  val form: Form[BigDecimal] = formProvider(BigDecimal(99999999.99))
+  val form: Form[BigDecimal] = formProvider(Some(BigDecimal(99999999.99)))
```
Comment (liampan) [inline test/forms/ConvertedPackagingCreditFormProviderSpec.scala:29]: this covers both branches of testing an should still pass :D 
```suggestion
  val form: Form[BigDecimal] = formProvider(None)
```

**Review by liampan — APPROVED** [review]

File: test/forms/ConvertedPackagingCreditFormProviderSpec.scala (hunk 3 above)
Comment (lmcewanhmrc) [reply test/forms/ConvertedPackagingCreditFormProviderSpec.scala:29]: Seems happy

**Pull request merged**